.github/workflows/deploy.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches:
       - main
       - dev
+      - test_teal_slice_version@main
```

Comment:
```suggestion
```

Reply:
Will remove this before merge :)

Reply:
Why is it needed while in the code below it takes directly from the commit?

```

                  if (package$Package == "teal.slice" && "${{ matrix.channel }}" == "stable") {
                    renv::record("insightsengineering/teal.slice@103491c67e26a9a85d636bd105a806904a0111ec")
```

Reply:
`test_teal_slice_version@main` is the current branch name, Needed this to test the deployment in this branch because they only run on `main` and `dev`
Yes, the commit hash makes sure that we install `teal.slice v0.4.0.9020` with the fix.

Reply:
makes sense now.